estuary_updater/handlers/distgit.py

```diff
@@ -89,3 +106,20 @@ def handle(self, msg):
 
         branch.contributors.connect(author)
         branch.commits.connect(commit)
+
+    def distgit_push_handler(self, msg):
```

Comment:
Optional: Since the class is called `DistGitHandler`, prefixing the methods with `distgit_` is somewhat redundant. The prefix could be removed.